appengine/flexible/scipy/requirements.txt

```diff
@@ -0,0 +1,4 @@
+Flask==0.11.1
+gunicorn==19.6.0
+scipy==0.18.0
+Pillow==3.3.1
```

Comment:
nit: Is Pillow necessary? I assume it's just a dependency of scipy, since I don't see it imported itself?